front/lib/plans/plan_codes.ts

```diff
@@ -0,0 +1,12 @@
+// Current free plans:
+export const FREE_UPGRADED_PLAN_CODE = "FREE_UPGRADED_PLAN";
```

Comment:
Would be nice to replace all occurrences of these strings in the codebase?

We can't really type the planCode since we'll have arbitrary DB ones so relying on such variables for these plan codes is indeed good hygiene. Let's make sure that it's used everywhere :+1: